Solver_interface/doc/Solver_interface/Concepts/SparseLinearAlgebraTraits_d.h

```diff
@@ -143,18 +152,18 @@ typedef unspecified_type NT;
 
 /// @}
 
-/// \name Creation 
+/// \name Creation
 /// @{
 
 /*!
 Create a square matrix initialized with zeros.
 */
-Matrix(int dimension);
+Matrix(Index dimension);
```

Comment:
Index not defined in Matrix.

Reply:
Can we address that when we solve the more general Issue #3479 ?

Reply:
Then only put it in the traits concept.

Reply:
As it is a `SparseLinearAlgebraTraits_d::Matrix`,  why does it then not "see" `SparseLinearAlgebraTraits_d::Index` ?

Reply:
I just added it locally in the scope of the `Matrix` concept.

Reply:
Where?